web/templates/settings/organization_permissions_admin.hbs

```diff
@@ -16,13 +16,18 @@
                       prefix="id_"
                       is_checked=realm_invite_required
                       label=admin_settings_label.realm_invite_required}}
-                    <label for="realm_invite_to_realm_policy" class="dropdown-title">{{t "Who can invite users to this organization" }}
+                    <label for="realm_invite_to_realm_policy" class="dropdown-title">{{t "Who can send email invitations to new users" }}
                     </label>
                     <select name="realm_invite_to_realm_policy" id="id_realm_invite_to_realm_policy" class="prop-element settings_select bootstrap-focus-style" data-setting-widget-type="number">
                         {{> dropdown_options_widget option_values=invite_to_realm_policy_values}}
                     </select>
                 </div>
 
+                {{> ../dropdown_widget_with_label
+                  widget_name="realm_create_multiuse_invite_group"
+                  label=(t 'Who can create reusable invitation links')
+                  value_type="number"}}
```

Comment:
I think this is being rendered a little differently than the other settings ...

<details>
<summary>screenshot as Iago</summary>

![Screenshot from 2023-08-10 15-41-14](https://github.com/zulip/zulip/assets/63245456/ec0db657-6adf-4126-aa87-d2bef86a3df5)
</details>

Reply:
Yeah we use the custom dropdown widget, like the ones used for `New stream announcements` setting for group-based settings. And other settings used the default `select` element.

Reply:
And the setting should be disabled for `Iago`. It is disabled for me.

Reply:
Yeah, earlier the UI of dropdown widget was not consistent with the other select fields. I made changes in last push and now it looks consistent.

Reply:
Pulled up the latest changes and the color looks good now, but the down arrow/chevron in the new setting dropdown is still a bit bigger than the other ones (at least in Chrome and Firefox on Linux) ...

<details>
<summary>Organization settings - Joining the organization - Iago</summary>

![Screenshot from 2023-08-14 14-41-29](https://github.com/zulip/zulip/assets/63245456/c7ca91b6-ce49-4f8e-a52b-20478b9858dd)
</details>

Reply:
This is a known issue with the component in general and we can deal with separately.